src/components/anti-everyone.ts

```diff
@@ -31,9 +43,63 @@ export default class AntiEveryone extends BotComponent {
         if (message.content.match(failed_everyone_re) != null) {
             // NOTE: .toLocaleString("en-US") formats this number with commas.
             const memberCount = this.wheatley.TCCPP.members.cache.size.toLocaleString("en-US");
-            await message.reply({
+
+            // Store the reply for later deletion, if necessary
+            if (!this.replies.has(message.author)) {
+                this.replies.set(message.author, []);
+            }
+            const reply = await message.reply({
                 content: `Did you really just try to ping ${memberCount} people?`,
             });
+
+            // Store the reply for later deletion, along with the message it was replying to
+            this.replies.get(message.author)!.push({ reply_to: message.id, reply });
```

Comment:
Could you `unwrap` here? (though it's basically guaranteed to be ok, unwrap will help catch future issues)